### DISCUSSION TITLE
Examples without TerriaMap

### DISCUSSION THREAD

**Discussion by jonathanpascoe** [TerriaJS/terriajs#6457] [Q&A]
I was just curious if anyone has had success using TerriaJS without TerriaMap? As in, a direct integration into an existing app. Are there any guides, examples, or suggestions?

Thanks in advance for the support.

**Comment by tephenavies** [comment]
Hi @jonathanpascoe. At the moment TerriaJS is coupled tightly to TerriaMap. TerriaMap has the webpack build configuration and some React components necessary to run TerriaJS. What might be possible is iframing a TerriaMap within another app and controlling it with messages (see https://docs.terria.io/guide/deploying/controlling-in-an-iframe-or-popup/)

**Reply by jonathanpascoe** [reply]
I already have the iframe implementation in place. I was hoping to evolve that a bit if possible. Thanks for the support @steve9164.
